pipeline/train/train.py

```diff
@@ -20,6 +20,9 @@
 train_dir = Path(__file__).parent
 
 
+CJK_LANGS = ["zh", "ja", "ko"]
```

Comment:
Thought: I guess we could want to put other languages in here that would use this same type of strategy, so CJK_LANGS could be other things. This is probably fine.

Reply:
yeah, probably we can expand that in the future and rename